app/design/frontend/Magento/luma/Magento_Catalog/web/css/source/module/_toolbar.less

```diff
@@ -170,7 +170,6 @@
     }
 
     .modes {
-        display: inline-block;
```

Comment:
could you explain why it was removed?

Reply:
When use float element will treated as block element so basically display define will unnecessary

Reply:
Updated change here since previously remove line will cause missing modes switch toolbar